package.json

```diff
@@ -20,6 +20,9 @@
     "updateOncoKbAPI": "npm run fetchOncoKbAPI && npm run buildOncoKbAPI",
     "fetchOncoKbAPI": "curl http://oncokb.org/api/v1/v2/api-docs > src/shared/api/generated/OncoKbAPI-docs.json",
     "buildOncoKbAPI": "node scripts/generate-api.js src/shared/api/generated OncoKbAPI",
+    "updatePdbAnnotationAPI": "npm run fetchPdbAnnotationAPI && npm run buildPdbAnnotationAPI",
+    "fetchPdbAnnotationAPI": "curl https://cbioportal.mskcc.org/pdb-annotation/v2/api-docs?group=pdb_annotation > src/shared/api/generated/PdbAnnotationAPI-docs.json",
+    "buildPdbAnnotationAPI": "node scripts/generate-api.js src/shared/api/generated PdbAnnotationAPI",
```

Comment:
To get PDB data, we use a separate service (pdb-annotation) running on cBioPortal servers.